src/islet/islet_shim.cc

```diff
@@ -26,22 +26,22 @@ using namespace certifier::utilities;
 static const char CLAIM_TITLE_USER_DATA[] = "User data";
 static const char CLAIM_TITLE_RIM[] = "Realm initial measurement";
 
-bool cca_Init(const int cert_size, byte *cert) {
+bool islet_Init(const int cert_size, byte *cert) {
   return true;
 }
 
-bool cca_Attest(const int what_to_say_size, byte* what_to_say,
+bool islet_Attest(const int what_to_say_size, byte* what_to_say,
```

Comment:
There is a little bit of very close names with this change. 

The new name `islet_Attest()` is very close to the actual API, `islet_attest()`; just the upper-case `A` is the diff.

Same issue applies to other interface functions.

But I can't do much about it w/o reworking the naming scheme all over.

If you can live with this naming closeness, ... I suggest we leave it alone for now.